Makefile

```diff
@@ -307,6 +307,10 @@ py_test_files := \
 # - 50664 ipwidgets - User Jupyter. Min ver 5.2.2 to 8.0.0. Sanitize descriptions
 # - 50463 ipwidgets - from 5.2.2 to 8.0.0.rc2
 # - 50792 nbconvert - from 5.0.0 to 6.5.1
+# - Oct 2022
+# - 51159 cryptography  v 39.0.0 drops support for C library "LibreSSL" < 3.4
```

Comment:
Should cryptography not be upgraded in the minimum-constraints and requirements files, where possible?